.github/workflows/runners.yaml

```
@@ -77,9 +77,9 @@ jobs:
           tags: |
             ${{ env.DOCKERHUB_USERNAME }}/${{ matrix.name }}:v${{ env.RUNNER_VERSION }}-${{ matrix.os-name }}-${{ matrix.os-version }}
             ${{ env.DOCKERHUB_USERNAME }}/${{ matrix.name }}:v${{ env.RUNNER_VERSION }}-${{ matrix.os-name }}-${{ matrix.os-version }}-${{ env.sha_short }}
-            ${{ env.DOCKERHUB_USERNAME }}/${{ matrix.name }}:latest
-            ghcr.io/${{ github.repository }}/${{ matrix.name }}:latest
+            ${{ env.DOCKERHUB_USERNAME }}/${{ matrix.name }}:${{ matrix.os-name }}-${{ matrix.os-version }}-latest
```

```suggestion
            ${{ env.DOCKERHUB_USERNAME }}/${{ matrix.name }}:${{ matrix.os-name }}-${{ matrix.os-version }}
```

This looks geat overall! Thanks for your efforts. One suggestion for this though- Can we omit the "-latest" suffix for this variant? I think we are basically omitting that suffix also on the existing `v${{ env.RUNNER_VERSION }}-${{ matrix.os-name }}-${{ matrix.os-version }}` image.

---

We are that's true. Now you've pointed it out it's a bit annoying but I think we probably should stick with not having it for backwards compatibility reasons.